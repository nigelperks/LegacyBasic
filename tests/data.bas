10 DATA 59.1,"  message  ",house,  houses  ,  3.14
20 READ A:PRINT A
30 READ A$:PRINT ">"A$"<"
40 READ A$:PRINT ">"A$"<"
50 READ A$:PRINT ">"A$"<"
60 READ A$:PRINT ">"A$"<"
70 READ X$:PRINT X$
80 DATA "FINAL"
90 RESTORE
100 READ A:PRINT A
