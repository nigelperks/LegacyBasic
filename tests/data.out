 59.1 
>  message  <
>house<
>houses<
>3.14<
FINAL
 59.1 
